Client/core/CModManager.cpp

```diff
@@ -347,7 +341,7 @@ void CModManager::InitializeModList(const char* szModFolderPath)
         VerifyAndAddEntry(szModFolderPath, ToUTF8(FindData.cFileName));
 
         // Search until there aren't any files left
-        while (FindNextFileW(hFind, &FindData) == TRUE)
+        while (FindNextFileW(hFind, &FindData) == true)
```

Comment:
`FindNextFileW` does not return a C++ `bool` but a Win32 `BOOL`, which is different.